src/trex/dataset.py

```diff
@@ -37,13 +46,15 @@ def read_one(self, path, output_bam_path, cell_id_tag="CB", require_umis=True):
             require_umis,
             cell_id_tag,
         )
-        umis = list()
+        '''
```

Comment:
I see this code was moved to a different method. You should therefore remove this commented-out code.